network_diffusion/models/mic_model.py

```diff
@@ -16,75 +33,99 @@
 
 
 class MICModel(BaseModel):
-    """This model implements Multilayer Independent Cascade Model.
-
-    The model is extended by Multilayer
-    """
+    """This model implements Multilayer Independent Cascade Model."""
 
     INACTIVE_NODE = "0"
     ACTIVE_NODE = "1"
-    PROCESS_NAME = "ICM"
     ACTIVATED_NODE = "-1"
-    probability = 0.0
+    PROCESS_NAME = "MICM"
 
     def __init__(
         self,
-        seeding_budget: Tuple[NumericType, NumericType],
+        seeding_budget: Tuple[NumericType, NumericType, NumericType],
```

Comment:
You sending budget on 3 states. It is required. It means 2 states are not enough. Maybe you take in account the next epochs when this state is

Reply:
I comprehend a seeding budget as information about percentages of agents in each state defined for the process. For instance, we can have: [32,68,0] -> 32% are inactive, 68% are active, and 0% are activated before the simulation